README.md

```diff
@@ -43,6 +43,10 @@ To suggest your contribution to the Keptn Contrib organization, it has to fulful
 * Apply Roles, ClusterRoles, Rolebindings and ClusterRoleBindings with a minimum set of useful permissions on that service account
 * Prefer namespaced Roles and Rolebindings over ClusterRoles and ClusterRoleBindings
 
+As references, you can take a look 
+how the the `keptn-default` Service Account is used in the `unleash-service` [here](https://github.com/keptn-contrib/unleash-service/pull/10/files)
```

Comment:
could you please change to "... take a look at how the ... " (remove the duplicated the)

Reply:
Thx, done